### ISSUE TITLE
Merge into CodeMirror?

### ISSUE THREAD

**Issue by gkoberger** [mkaminsky11/swift-codemirror#2]
This would be great if it was actually in the codemirror repo. Much easier to install and set up :)

**Comment by mkaminsky11** [comment]
yeah, I'll do that soon. I kind of took a break on the project...it's not quite done.
